UnityProject/Assets/Scripts/Messages/Server/SpritesMessages/SpriteUpdateMessage.cs

```diff
@@ -367,7 +376,7 @@ public static void GenerateSerialisation(SpriteHandlerManager.SpriteChange sprit
 				ToReturn.Append(Convert.ToChar(Mathf.RoundToInt(Colour.g * 255)));
 				ToReturn.Append(Convert.ToChar(Mathf.RoundToInt(Colour.b * 255)));
 				ToReturn.Append(Convert.ToChar(Mathf.RoundToInt(Colour.a * 255)));
-				
+
 			}
```

Comment:
you forgot to add it to the Serialiser,

Reply:
very confused as to how this was working when i built tested it :p